examples/l2services/l2services.yaml

```diff
@@ -85,4 +94,5 @@ metadata:
   name: l2service-twenty
 spec:
   SegmentationID: 20
+  Subnets: ["ipv6-subnet-twenty", "ipv4-subnet-twenty"]
```

Comment:
Should we have an example only with ipv6 ? I know currently with whereabouts it would pick ipv6 only and not v4,in case we support another cni.

Reply:
The reason that I putted that way is because I would like to be generic. So here Whereabouts will pick the first item from the list. But if we had dualstack will pick both. the reason that I putted that way is because I would like to show that we intent to support dual stack.

Reply:
Of course this is not major point for discussion. So I think we can keep it this way for now.